fotoapparat/src/main/java/io/fotoapparat/FotoapparatBuilder.java

```diff
@@ -144,6 +145,14 @@ public FotoapparatBuilder sensorSensitivity(@NonNull SelectorFunction<Range<Inte
     }
 
     /**
+     * @param jpegQuality of the picture (1-100)
+     */
+    public FotoapparatBuilder jpegQuality(@NonNull Integer jpegQuality) {
```

Comment:
Let's add `@IntRange` annotation for more robustness

Reply:
That's a great idea! 
done in https://github.com/Fotoapparat/Fotoapparat/pull/125/commits/41bec5dc0ac4168d16e7e0525d93354ebef223a9